src/components/Mutation.tsx

```diff
@@ -20,7 +20,8 @@ interface MutationHandlerState {
 
 interface MutationChildProps extends MutationHandlerState {
   executeMutation: <T = any, V = object>(
-    variables?: V
+    variables?: V,
+    skip?: boolean
```

Comment:
it doesn't really make sense to add skip to mutations as they're always methods that are called imperatively